packages/next-frontend/src/components/create-campaign/form/dex-step/styles.module.css

```diff
@@ -0,0 +1,42 @@
+.amm_step__dex__preview {
```

Comment:
Same as above

Reply:
Fixed in 553b93b44f72fa608f10241ebc55277f6270e045.